### PR TITLE
Do not restore on PostgreSQL TimescaleDB

### LINKED ISSUE #1555: [TimescaleDB] Explain the importance of not doing a database restore on a PostgreSQL using TimescaleDB
To restore a backup, users have to ask on the support.

### PR DESCRIPTION
**Pull request by curzolapierre**
Do not merge, wait for https://github.com/Scalingo/documentation/pull/1520 to be merged before.

Fix #1555

### CODE REVIEW

File: src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Please, specify in your support request the timestamp of the backup.
+
+## Warnings
+
+Several points are important to take into account.
+
+It may happen some trouble due to a PostgreSQL and TimescaleDB version mismatch
+between the time the backup was made and the database you are trying to restore to.
+
+You cannot backup an individual hypertable, `pg_dumps` creates dumps that do not
```
Comment (EtienneM) [inline src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md:37]:
```suggestion
You cannot backup an individual hypertable, `pg_dump` creates dumps that do not
```

File: src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+The backup restoration on TimescaleDB is requiring a preparation of the PostgreSQL database
+which needs admin rights. That's why you have to ask our support to handle the process.
+Please, specify in your support request the timestamp of the backup.
+
+## Warnings
+
+Several points are important to take into account.
+
+It may happen some trouble due to a PostgreSQL and TimescaleDB version mismatch
+between the time the backup was made and the database you are trying to restore to.
```
Comment (EtienneM) [inline src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md:35]: This sentence is unclear and should be rephrased

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+{% warning %}
+  The backup restoration process may put the base in an undesirable state.
+  Pay attention to the restoration process.
+{% endwarning %}
+
+## Dump
+
+The only dump available of a PostgreSQL using the TimescaleDB extension are the
+[on demand backups]({% post_url databases/postgresql/2000-01-01-start %}#on-demand-backups).
```
[the remark this answers is not in the document]
Comment (curzolapierre) [reply src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md:17]: It is possible to execute `pg_dump` but you cannot use it as a backup restore on a TimescaleDB hosted on Scalingo.

File: src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+title: TimescaleDB
+nav: Dump and restore
+modified_at: 2022-02-18 00:00:00
+tags: timescale databases postgresql extensions dump restore
+index: 2
+---
+
+{% warning %}
+  The backup restoration process may put the base in an undesirable state.
+  Pay attention to the restoration process.
```
[the remark this answers is not in the document]
Comment (EtienneM) [reply src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md:11]: Then shouldn't we remove it? ^^

```suggestion
```

File: src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md (hunk 2 above)
Comment (EtienneM) [reply src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md:48]: I'm sorry but I really don't understand this sentence. Can you explain in French what you want to explain here?

File: src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (curzolapierre) [reply src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md:11]: Oops a forget yes ^^

File: src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md (hunk 2 above)
Comment (curzolapierre) [reply src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md]: It is clearer to refer to the doc with all the details.

**Review by EtienneM — APPROVED** [review]

**Review by EtienneM — APPROVED** [review]

**Pull request merged**